scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -527,20 +532,48 @@ <h5 class="modal-title" id="archiveCaseLabel">Confirm archive case</h5>
 </div>
 {% endmacro %}
 
-{% macro delivery_report(institute, case, date=None) %}
+{% macro analysis_report(institute, case, analysis) %}
 <div href="#" class="bg-dark list-group-item text-white">
   <div class="d-flex flex-row flex-fill bd-highlight">
     <div>
-      <span class="menu-collapsed ml-2">Delivery ({{date or case.analysis_date.date() }})</span>
-    </div>
-    <div>
-      <a href="{{ url_for('cases.delivery_report', institute_id=institute,
-                        case_name=case.display_name, date=date) }}" target="_blank"><i class="fa fa-link"></i></a>
+      <span class="menu-collapsed ml-2">Delivery ({{ analysis.date.date() }})</span>
     </div>
+    {% if analysis.delivery_report != case.delivery_report and analysis.delivery_report is existing %}
+      <div>
+        <a href="{{ url_for('cases.delivery_report', institute_id=institute,
+                          case_name=case.display_name, date=analysis.date.date()) }}" target="_blank"><i class="fa fa-link"></i></a>
+      </div>
+      <div>
+        <a href="{{ url_for('cases.delivery_report', institute_id=institute,
+                          case_name=case.display_name, date=analysis.date.date(), format='pdf') }}" download><i class="fa fa-file-pdf-o"></i></a>
+      </div>
+    {% elif analysis.delivery_report == case.delivery_report %}
+      <div><i class="fa fa-exclamation-circle ml-1" data-toggle="tooltip" title="A delivery report had been registered for the analysis but it is the same as the latest."></i></div>
```

Comment:
I think it's my fault that you changes the code and now it shows duplicated report dates (the test I did before). I think it was good as it was before, where if the date of the analysis is the same as the date of the case.delivery report, then the analysis is not shown. Can we revert?

Reply:
Last things first, on what browser and so on? It works on safari.

![Screenshot 2022-02-10 at 14 19 57](https://user-images.githubusercontent.com/758570/153416682-861d12ff-ee45-40b4-a9d8-32a7f458a23d.png)

Reply:
I starting to think the db may have some kinks since old cg uploads, and that part of the old operation was to deal with both that and sort of the way case re-upload works from the cli. But, let's check!

Reply:
Chrome, tested once more and I don't see the popup, while for instance in this other case it works: https://scout-stage.scilifelab.se/cust003/20192

Reply:
> I starting to think the db may have some kinks since old cg uploads, and that part of the old operation was to deal with both that and sort of the way case re-upload works from the cli. But, let's check!

If you dig too dip I'm afraid of what you could find.. 😄

Reply:
> Chrome, tested once more and I don't see the popup, while for instance in this other case it works: https://scout-stage.scilifelab.se/cust003/20192

Erratic issue then? My Chrome says its updated and has the tooltip. And you are sure you hovered for a little bit: it's a tooltip as requested, not a popover that you click.

![Screenshot 2022-02-10 at 14 34 52](https://user-images.githubusercontent.com/758570/153418866-3e12dc26-7cad-40ae-b4a5-eba27e3b00b7.png)

![Screenshot 2022-02-10 at 14 34 11](https://user-images.githubusercontent.com/758570/153418863-774ea6c3-2e01-4197-bd92-8099b949c6b3.png)

Reply:
> Chrome, tested once more and I don't see the popup, while for instance in this other case it works: https://scout-stage.scilifelab.se/cust003/20192

I can't see the popups in Safari (v. 14.0.3)

Reply:
Both on Safari and Chrome, [this case](https://scout-stage.scilifelab.se/cust003/15001):I'm not clicking, I'm doing the same thing as when I see the popups for other cases. Just in this case they don't work

Reply:
Justhusky has seven potential analyses.
![Screenshot 2022-02-10 at 14 16 08](https://user-images.githubusercontent.com/758570/153417715-d63edc9d-a81c-4327-98ab-8085da35d779.png)
"Only" four actually provided reports. Three are unique, and one was a re-upload of the latest. I think this is correct, and no information is lost. But, it will very rarely hurt to conceal the duplicates if they are same date and report. That allows for some butter fingers on upload without conveying much confusion. More than possibly some users complaining about their comments being marked "old".

Reply:
> 
> ![Screenshot 2022-02-10 at 14 34 52](https://user-images.githubusercontent.com/758570/153418866-3e12dc26-7cad-40ae-b4a5-eba27e3b00b7.png)

My version: Version 98.0.4758.80 (Official Build) (x86_64) 🤔

Reply:
👀 That is the same case as you posted before, and the same case still works just fine here, with both Safari, Firefox and Chrome. I don't know, perhaps check fire up inspector and see if you have like any lingering script issues or sth?

Reply:
It's fine, let's disregard the missing popups

Reply:
Ok, Ive read a few of the old PRs now. I still think the current is the highest fidelity reproduction of what is in the database, assuming the upload code was always the way it was. But to cater for possible old issues, and also give some butter finger leeway on re-upload handling, I think the justhusky comment above also holds. Just hiding the same-as-case-analysis_date analysis_reports is not loosing much information, while avoiding some user worries.